test/pytest/test_batchnorm_pytorch.py

```diff
@@ -13,15 +13,24 @@
 atol = 5e-3
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture
 def data():
     np.random.seed(0)
     X = np.random.rand(100, in_shape)
     return X
 
 
+@pytest.fixture
+def fusion_data():
+    n_batch = 2
+    n_in = 2
+    size_in_height = 32
+    X = np.random.rand(n_batch, n_in, size_in_height)
+    return X
+
+
 @pytest.mark.parametrize('io_type', ['io_parallel', 'io_stream'])
-@pytest.mark.parametrize('backend', ['Vivado', 'Vitis', 'Quartus', 'Catapult'])
```

Comment:
Looks like I accidentally committed this change. Can you undo it @sei-rquartiano ?